test/javascript/packs/utils/calendar.test.js

```diff
@@ -24,4 +25,48 @@ describe("calendar", () => {
       });
     });
   });
+
+  describe("prev", () => {
+    const base_date = moment('2022-08-01').utc()
+    it('returns previous month in same day', () => {
+      const prev_month = prev(base_date)
+      expect(prev_month.format('YYYY-MM-DD')).toBe('2022-07-01')
+    });
+  });
```

Comment:
In JS we don't use Snake Case:
```suggestion
  describe("prev", () => {
    const baseDate = moment('2022-08-01').utc()
    it('returns previous month in same day', () => {
      const prevMonth = prev(baseDate)
      expect(prevMonth.format('YYYY-MM-DD')).toBe('2022-07-01')
    });
  });
```